service/launchd.go

```
@@ -108,11 +109,99 @@ func (j *Launchd) reloadService() error {
+				}
+			}
+		}
+
+		// Create new arguments list
+		newArgs := make([]string, 0)
+
+		i := 0
+		for ; i < len(programArgs); i++ {
+			if strings.HasPrefix(programArgs[i], "--home=") {
+				newArgs = append(newArgs, programArgs[i])
+				i++
```

golangci-lint (1.62.2)

ineffectual assignment to i

(ineffassign)